src/CovidSim.cpp

```diff
@@ -2925,7 +2925,7 @@ int RunModel(int run, std::string const& snapshot_save_file, std::string const&
 								do
 								{
 									l = (int)(((double)P.PopSize) * ranf()); //// choose person l randomly from entire population. (but change l if while condition not satisfied?)
-								} while ((abs(Hosts[l].inf) == InfStat_Dead) || (ranf() > P.FalsePositiveAgeRate[HOST_AGE_GROUP(l)]));
+								} while ((abs((int) Hosts[l].inf) == (int) InfStat::Dead) || (ranf() > P.FalsePositiveAgeRate[HOST_AGE_GROUP(l)]));
 								DoFalseCase(l, t, ts, 0);
```

Comment:
Possibly Hosts[l].inf == one of {InfStat::Dead_WasAsymp, InfStat::Dead_WasSymp, InfStat::Dead}

Reply:
OK, happy to withdraw the change request, as long as the abs code is addressed in the next PR. (unless you want to address it in this PR.)

Reply:
I'd replace abs in this PR. It's going to touch a lot of the same lines.